examples/Continuous_Beam/Program.cs

```diff
@@ -0,0 +1,642 @@
+                }
+                #endregion
+
+                // printout parts list
+                #region Generate parts list
+                model.generate_parts_lists();
+                parts_list_all_by_material_row[] partListByAllMaterial = model.get_parts_list_all_by_material();
+                foreach (var item in partListByAllMaterial)
+                {
+                    if (!item.description.Contains("Total:"))
+                    {
+                        Console.WriteLine("Material no: {0}\t Material name: {1}\t object type: {2}\t coating:{3}\t volume: {4}\t mass: {5}", item.description, item.row.material_name, item.row.object_type, item.row.total_coating, item.row.volume, item.row.mass);
+                    }
+                    else
+                    {
+                        Console.WriteLine("Material total\t \t \t coating:{0}\t volume: {1}\t mass: {2}", item.row.total_coating, item.row.volume, item.row.mass);
+                    }
+                }
+
+                Console.WriteLine("Members: ");
+                parts_list_members_by_material_row[] partListMemberByMaterial = model.get_parts_list_members_by_material();
+                foreach (var item in partListMemberByMaterial)
+                {
+                    if (!item.description.Contains("Total"))
+                    {
+                        Console.WriteLine("Material no: {0}\t Material name: {1}\t section: {2}\t members no:{3}\t quantity: {4}\t length: {5}\t unit surface area: {6}\t volume: {7}\t unit mass: {8}\t member mass: {9}\t total length: {10}\t total surface area: {11}\t total volume:{12}\t total mass:{13}",
+                        item.description, item.row.material_name, item.row.section_name, item.row.members_no, item.row.quantity, item.row.length, item.row.unit_surface_area, item.row.volume, item.row.unit_mass, item.row.member_mass, item.row.total_length, item.row.total_surface_area, item.row.total_volume, item.row.total_mass);
+                    }
+                    else
+                    {
+                        Console.WriteLine("Total \t \t \t \t quantity: {4}\t length: {5}\t unit surface area: {6}\t volume: {7}\t unit mass: {8}\t member mass: {9}\t total length: {10}\t total surface area: {11}\t total volume:{12}\t total mass:{13}",
+                                            item.description, item.row.material_name, item.row.section_name, item.row.members_no, item.row.quantity, item.row.length, item.row.unit_surface_area, item.row.volume, item.row.unit_mass, item.row.member_mass, item.row.total_length, item.row.total_surface_area, item.row.total_volume, item.row.total_mass);
+                    }
+                }
+                #endregion
+
+                //save the model before closing
+                model.save("C:\\Users\\GoebelR\\Documents\\Webservices\\testmodels\\testmodel");
```

Comment:
Don't use absolute path

Reply:
I implemented that change into the first pull request